iceoryx_examples/singleprocess/CMakeLists.txt

```diff
@@ -39,6 +39,13 @@ set_target_properties(single_process PROPERTIES
 )
 target_compile_options(single_process PRIVATE ${ICEORYX_WARNINGS} ${ICEORYX_SANITIZER_FLAGS})
 
+# Since we have one process where multiple threads are created we need to increase the stack size for some OS types
+if(QNX)
+    target_link_options(single_process BEFORE PRIVATE -Wl,-z,stack-size=3500000)
```

Comment:
Could you please create a cmake variable at the top named like `ICEORYX_SINGLE_PROCESS_STACK_SIZE` and use it here. Then we can adjust it more easily if we ever have to and do not forget something.